src/compositions/usePagination.ts

```diff
@@ -76,7 +77,7 @@ export function usePagination<
   const pages = computed(() => Math.ceil(total.value / getLimit()))
 
   const countSubscriptionParameters = computed(() => {
-    if (page.value) {
+    if (isDefined(page.value)) {
```

Comment:
I could use a gut check on this one - I don't quite understand what's up here but it seems like this composition has a weird catch-22 where infinite mode sets the page to 0, which causes this parameters computed to exit early and so the fetch functions never run. I'm not sure why we've not seen this before, I suspect it's something of a race condition?

Reply:
🧐 infinite mode should coalesce the page getter to 0 which would have this return null to disable the count subscription. 

Hmmm maybe open a separate issue to track and not block this stuff?

Reply:
I'm gonna dig a little more - without this change I couldn't get any of my filter calls to run until I updated one of the filters.